[nltk_data] Error loading stopwords: <urlopen error Remote end closed
[nltk_data]     connection without response>
[nltk_data] Error loading punkt: <urlopen error Remote end closed
[nltk_data]     connection without response>
/home/cse/dual/cs5190443/scratch/anaconda3/envs/a100s2/lib/python3.9/site-packages/torch/nn/modules/transformer.py:282: UserWarning: enable_nested_tensor is True, but self.use_nested_tensor is False because encoder_layer.self_attn.num_heads is odd
  warnings.warn(f"enable_nested_tensor is True, but self.use_nested_tensor is False because {why_not_sparsity_fast_path}")

[Timed Out]